test/integration/operatingsystem_js_test.rb

```diff
@@ -4,4 +4,24 @@ class OperatingsystemJSTest < IntegrationTestWithJavascript
   test "index page" do
     assert_index_page(operatingsystems_path, "Operating Systems", "Create Operating System")
   end
+
+  test "delete operating system" do
+    visit operatingsystems_path
+    first_row = find("table > tbody > tr:nth-child(1)")
+    assert first_row.find("td:nth-child(1)").text, "centos 5.3"
+    assert has_no_css?("#app-confirm-modal")
+
+    actions = first_row.find("td:nth-child(3) > div")
+    actions.find("a.dropdown-toggle").click
+    actions.find("ul > li > a.delete").click
+
+    confirm_modal = page.find("#app-confirm-modal")
+    assert confirm_modal.find(".pf-c-modal-box__title-text").text, "Confirm"
+    assert confirm_modal.find(".pf-c-modal-box__body").text, "Delete centos 5.3?"
+
+    confirm_button = confirm_modal.find("footer > button:nth-child(1)")
+    assert confirm_button.text, "Confirm"
+    confirm_button.click
+    assert has_no_css?("#app-confirm-modal")
```

Comment:
@ezr-ondrej @tbrisker @amirfefer maybe you can help with the test?
I need to wait somehow for the browser to reload after the confirm click
and test if the OS was successfuly deleted

Reply:
Thanks, pushed another commit fixing the test - the problem was that the OS wasn't deleted because it was still in use. Adding `DEBUG_JS_TEST=1` to the test run causes it to run in chrome instead of headless chrome so you can see what it is doing - so I saw it was displaying an error message. Added a new OS with name `aaa` to make sure it is listed first, and also fixed the `assert` that should have been `assert_equal` (`assert` only checks if the value of the first argument is truthy, and if it isn't it displays the second argument as message).